test/specs/post.js

```diff
@@ -9,6 +9,9 @@ var post = proxyquire('../../src/post', {
 
 var pkg = {
   version: '1.0.0',
+  publishConfig: {
+    tag: 'beta'
+  },
```

Comment:
Don’t change an existing test, please create a new on

Reply:
I was about to, but it raises the error I mentioned even without the changes. What do you recommend to fix that?

Reply:
the tests should pass, they just did about 19h ago on travis:
https://travis-ci.org/semantic-release/semantic-release/builds/275739048

The tests that currently fail are related to your changes.

Before the changes, what errors do you see?

Reply:
Hmmm that's weird. I forked and just ran npm install and npm test. Do I need anything else?

Reply:
Log: https://pastebin.com/HARMnXg5

Reply:
try to run in node 8, we recently deprecated support for lower versions via https://github.com/semantic-release/semantic-release/pull/414

I think you also need to setup CouchDB locally for the integration tests to work: http://couchdb.apache.org/

Reply:
Alright, just updated to node 8.5 and npm to 5.3 and it seems to be the cause. It still raises the following error though: https://pastebin.com/PPDfSnz3

Reply:
have you installed CouchDB? Can you run `couchdb` in your terminal?

Reply:
I've installed it, and it's running in http://localhost:5984/_utils/ but I'm not sure what else I need to set it up for the tests

Reply:
The whole setup is not ideal right now, I don’t even know if it’s working on Windows :( 

as a workaround, can you try to adapt these lines for local testing:

https://github.com/semantic-release/semantic-release/blob/88b5895f800f608eca3f1f4b435b34891d00082d/test/registry/start.sh#L22-L23

Change the port number in L22 to to `5984` and comment out the second. Make sure you created an admin account with username "admin" and password "password" at http://localhost:5984/_utils/

Reply:
Still doesn't seem to work. I will try with a different couchdb version tomorrow, as running couchdb is producing some weird output (kernel-poll not supported).

Reply:
I’m sorry for the trouble, thanks for your patience! If it doesn’t work, you can still push and have travis run the integration tests.

Reply:
It seems to be a problem with privileges in Windows... Still trying to figure out a solution.